src/flush-css-chunks-webpack-plugin.js

```diff
@@ -64,6 +65,10 @@ class FlushCSSChunks {
             return;
           }
 
+          if (this.options.entryOnly && this.options.entries && this.options.entries.every(entry => chunk.name !== entry)) {
```

Comment:
Maybe we should check here if the `entries` is array like `Array.isArray(this.options.entries)` and replace the `this.options.every(...)` with `!this.options.includes(chunk.name)`

Reply:
Will change to ```includes```.

What should we do if it is not an array? Throw error or just ignore it?

Reply:
I think we should throw and Error which "describes" wrong configuration/option property

Reply:
Switched to `includes` and added Error about wrong options